pkgs/development/python-modules/essentials-openapi/default.nix

```diff
@@ -0,0 +1,65 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  click,
+  essentials,
+  flask,
+  hatchling,
+  httpx,
+  jinja2,
+  markupsafe,
+  pydantic,
+  pytestCheckHook,
+  pythonImportsCheckHook,
+  pyyaml,
+  rich,
+  setuptools
+}:
+buildPythonPackage rec {
+  pname = "essentials-openapi";
+  version = "1.0.7";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Neoteroi";
+    repo = "essentials-openapi";
+    rev = "v${version}";
+    hash = "sha256-j0vEMNXZ9TrcFx8iIyTFQIwF49LEincLmnAt+qodYmA=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+  ];
+
+  nativeCheckInputs = [
+    flask
+    httpx
+    pydantic
+    pytestCheckHook
+    rich
+    setuptools
```

Comment:
```suggestion
```
According to https://github.com/Neoteroi/essentials-openapi/blob/4ffc7b1921835a22a386494c0acb4f4390f5f1a4/tests/common.py#L18, `pkg_resources` is only needed for Python 3.8.

Reply:
```
error: builder for '/nix/store/4annsxfapvaagljlz1vq7iawza6r1md5-python3.11-essentials-openapi-1.0.7.drv' failed with exit code 2;
       last 25 log lines:
       >     return _bootstrap._gcd_import(name[level:], package, level)
       > tests/test_v2.py:36: in <module>
       >     from tests.common import debug_result
       > tests/common.py:6: in <module>
       >     import pkg_resources
       > E   ModuleNotFoundError: No module named 'pkg_resources'
       > ______________________ ERROR collecting tests/test_v3.py _______________________
       > ImportError while importing test module '/build/source/tests/test_v3.py'.
       > Hint: make sure your test modules/packages have valid Python names.
       > Traceback:
       > /nix/store/y027d3bvlaizbri04c1bzh28hqd6lj01-python3-3.11.7/lib/python3.11/importlib/__init__.py:126: in import_module
       >     return _bootstrap._gcd_import(name[level:], package, level)
       > tests/test_v3.py:46: in <module>
       >     from tests.common import debug_result
       > tests/common.py:6: in <module>
       >     import pkg_resources
       > E   ModuleNotFoundError: No module named 'pkg_resources'
       > =========================== short test summary info ============================
       > ERROR tests/test_cli.py
       > ERROR tests/test_mk_v3.py
       > ERROR tests/test_v2.py
       > ERROR tests/test_v3.py
       > !!!!!!!!!!!!!!!!!!! Interrupted: 4 errors during collection !!!!!!!!!!!!!!!!!!!!
       > ============================== 4 errors in 0.25s ===============================
       > /nix/store/yq9qag5ak3w15nhcqplnhxpny6295l36-stdenv-linux/setup: line 1582: pop_var_context: head of shell_variables not a function context
```